code/drasil-example/projectile/lib/Drasil/Projectile/IMods.hs

```diff
@@ -97,7 +97,7 @@ landPosDerivSent2 = foldlSentCol [S "To find the", phrase landPos `sC`
   S "we want to find the", ch xPos, phrase value, sParen (ch landPos),
   S "at", phrase flightDur, fromSource timeIM]
 landPosDerivSent3 = foldlSentCol [S "From", refS speedIX,
-  sParen (S "with" +:+ E (defines iSpeed launSpeed)), S "we can replace", ch ixVel]
+  sParen (S "with" +:+ E (defines (sy iSpeed) (sy launSpeed))), S "we can replace", ch ixVel]
```

Comment:
why does `defines` now take symbols? this seems to be a (very small) regression?

Reply:
`defines` previously had an `Express` constraint for its first parameter's inputs. With iSpeed, it was a quantity and was `express`ed as a `sy iSpeed` in the end (the Express instance is just `express = sy` for all quantity-like chunks).  This is bringing up an issue with `defines`. I would've fixed it immediately because `defines` should always be in the form of `x $= ...expr...` but we have one area (QDefinitions) where we have symbols that represent functions, for which we want to have the LHS of the equation be a `x(a,b,c)` instead. I'll take a second look at the code to see if we can get this fixed in this branch quickly before a merge.

Reply:
For this particular instance of `defines`, what is going wrong is that we shouldn't be using this linguistic form inline at all. The relationship between `iSpeed` and `launSpeed` should be somehow captured in another piece of that, and the display form associated to that relationship should be what's used here.

So the first thing to do would be to fix those "bad" uses of `defines`. Then see those that are left.

Reply:
In this case, it looks like we're trying to substitute a variable with another in deriving a relation, `launSpeed` (launch speed) is an "instance" (of sorts -- I'm not quite sure what to call this relationship) of `iSpeed` (initial speed). Would this be a task for when we have refinements of models?

Related area in SRS (3rd sentence): https://jacquescarette.github.io/Drasil/examples/projectile/SRS/srs/Projectile_SRS.html#IM:calOfLandingDist

Reply:
what I'm wondering about is the source of that equality between `iSpeed` and `launSpeed`. It feels ad hoc to have it inline here. It feels like it ought to exist elsewhere, in dependently.

Reply:
I agree, I believe it was just a temporary measure. I was thinking that this would be a part of the refinement's system you mentioned a few meetings ago. I think this will be a rather large design needed before we can do something about it.

I'll make a ticket so that we can discuss it more.

Reply:
Moving discussion to #2852 so that we can more easily discuss and track progression of this.